src/tests/JittedMethodsCountingTest/JittedMethodsCounter/JittedMethodsCounter.cs

```diff
@@ -0,0 +1,77 @@
+// TODO: Add the Microsoft License header here.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Xunit;
+
+public class JittedMethodsCounter
+{
+    [Fact]
+    public static int TestEntryPoint()
```

Comment:
I'm not sure about this one, is that an earlier version of the test you're working on now so that we should ultimately get rid of it or are you envisioning two different modes of testing?

Reply:
This is going to be removed. It slipped through. The actual test is the other file you reviewed.